.github/workflows/go-compatible.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest]
-        go: [ '1.18' ,'1.20','1.21']
+        go: [ '1.17','1.18','1.19','1.20','1.21']
```

Comment:
You know 1.22 is the last one right? 😁

Reply:
That's exactly why it is not here, because this file only tests older go versions. Cutting edge go is tested in another file.
Glad you check the PR so carefully. Thanks a lot. 😊

Reply:
Maybe a comment just above the line became it's not obvious when seeing the line changed in a PR